packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -229,200 +487,122 @@ const npmInstall = (outputDir, {verbose}) => {
     }
 }
 
-// Validations
-const validProjectName = (s) => {
-    const regex = new RegExp(`^[a-zA-Z0-9-\\s]{1,${PROJECT_ID_MAX_LENGTH}}$`)
-    return regex.test(s) || 'Value can only contain letters, numbers, space and hyphens.'
-}
+/**
+ * Execute and copy the handlebars template to the output directory using
+ * the provided context object. If the file isn't a template, simply copy
+ * it to the destination.
+ *
+ * @param {string} inputFile
+ * @param {string} outputDir
+ * @param {Object} context
+ */
+const processTemplate = (relFile, inputDir, outputDir, context) => {
+    const inputFile = p.join(inputDir, relFile)
+    const outputFile = p.join(outputDir, relFile)
+    const destDir = outputFile.split(p.sep).slice(0, -1).join(p.sep)
```

Comment:
How about simplifying to this?
```suggestion
    const destDir = p.join(outputFile, '..')
```